File: materiais/Controle de Fluxo e Iteração/ifelse.c
#include <stdio.h>

// Exemplo de programa que avalia se
// um número digitado é par ou ímpar

int main(void)
{
	int num;
	printf("Digite um número: ");
	scanf("%d",&num);

	if(num % 2 == 0)
		printf("%d é um número par\n",num);
	else
		printf("%d é um número ímpar\n",num);
}

File: materiais/Controle de Fluxo e Iteração/Códigos/ifelse.py
# Exemplo de programa com controle de fluxo if...else
num = int(input("Digite um número: "))

if num > 0:
	print(f"{num} é positivo")
elif num == 0:
	print(f"Número é {num}")
else:
	print(f"{num} é negativo")